db/models/app.models.js

```diff
@@ -27,8 +27,18 @@ LEFT JOIN comments
 ON articles.article_id = comments.article_id
 GROUP BY 
 articles.article_id
-ORDER BY articles.created_at ASC;`)
+ORDER BY articles.created_at DESC;`)
     .then(({rows})=>{
         return rows
     })
+}
+
+exports.fetchCommentsByArticleId = (article_id) => {
+    return db
+    .query(`SELECT * FROM comments
+        WHERE comments.article_id = $1
+        ORDER BY comments.created_at DESC;`, [article_id])
+        .then(({rows})=>{
+            return rows
```

Comment:
To pass the suggested test, you will need to check the reason for the query returning empty rows:

- article exists but has no comments.
- article does not exist.

You will need to check whether the article exists in order to determine this. (You can almost certainly use an existing model for this check.)